optuna/samplers/_tpe/sampler.py

```diff
@@ -457,15 +472,33 @@ def sample_independent(
 
         return self._sample(study, trial, {param_name: param_distribution})[param_name]
 
+    def _get_params(self, trial: FrozenTrial) -> dict[str, Any]:
+        if trial.state.is_finished() or not self._multivariate:
+            return trial.params
+
+        params_strs = []
+        i = 0
+        while params_str_i := trial.system_attrs.get(f"{_RELATIVE_PARAMS_KEY}:{i}"):
+            params_strs.append(params_str_i)
+            i += 1
+
+        if len(params_strs) == 0:
+            return trial.params
+        else:
+            params_str = "".join(params_strs)
+            params = json.loads(params_str)
+            params.update(trial.params)
+            return params
```

Comment:
[nit] We can reduce the indents here.
```suggestion
        params_str = "".join(params_strs)
        params = json.loads(params_str)
        params.update(trial.params)
        return params
```